### PR TITLE
Mod4 branch

### PR DESCRIPTION
**Pull request by Valevach90**

### CODE REVIEW

File: mod2/src/main/java/org/example/ParserJson.java [hunk 1]
```diff
@@ -15,18 +15,23 @@
...
 
     public Configuration fromJSON() {
+
+        LOGGER.info("method fromJSON is start ");
         Configuration fileName = null;
         try {
             InputStream resourceAsStream = this.getClass()
                     .getClassLoader()
                     .getResourceAsStream("config.json");
+            LOGGER.info("configuration data is read" );
```
Comment (plisunov) [inline mod2/src/main/java/org/example/ParserJson.java]: Data is not readed in this line

File: mod2/src/main/java/org/example/RenameFiles.java [hunk 2]
```diff
@@ -1,17 +1,22 @@
...
+
 import java.io.File;
 
 public class RenameFiles {
+    private static final Logger LOGGER = Logger.getLogger(RenameFiles.class.getName());
     public static void renameFiles(Configuration config) {
         for (FileInfo files : config.getFiles()) {
             File oldFile = new File(files.getFileName());
             if (oldFile.exists()) {
+                LOGGER.info("files are renamed");
```
Comment (plisunov) [inline mod2/src/main/java/org/example/RenameFiles.java]: "File is exist. Rename started" message will be better.

File: mod2/src/main/java/org/example/RenameFiles.java [hunk 3]
```diff
@@ -1,17 +1,22 @@
...
                 String newFileName = oldFile.getParent() + config.getSuffix() + oldFile.getName();
                 File newFile = new File(newFileName);
                 oldFile.renameTo(newFile);
                 System.out.println(files.getFileName() + " > " + newFileName);
             } else {
+                LOGGER.info("files "+files.getFileName()+ " are not renamed");
```
Comment (plisunov) [inline mod2/src/main/java/org/example/RenameFiles.java:19]: This message is not correct.

File: mod2/src/main/resources/log4j.properties [hunk 4]
```diff
@@ -0,0 +1,12 @@
+# ������� �����������
+log4j.rootLogger=INFO, file
+
+# ������� ��� ������ � �������
+log4j.appender.file=org.apache.log4j.RollingFileAppender
+# ���� ��� ����� ����������� ��� ����
+log4j.appender.file.File=C:\\Users\\Dmitry\\IdeaProjects\\JavaUpSkill\\mod2\\src\\log\\log_file.log
```
Comment (plisunov) [inline mod2/src/main/resources/log4j.properties]: Can we use relative path for file? Also log file should not be placed to code path

**Review by plisunov — CHANGES_REQUESTED** [review]

**Review by Valevach90 — COMMENTED** [review]
последний вариант программы

**Pull request merged**